recipe/meta.yaml

```
@@ -13,14 +13,11 @@ source:
   - path: setup.py
 
 build:
-  number: 0
+  number: 1
```

Set this to 0 if we're not merging #1.
```suggestion
  number: 0
```